src/Manager.php

```diff
@@ -181,7 +181,7 @@ protected function buildRefreshClaims(Payload $payload)
             $persistentClaims,
             [
                 'sub' => $payload['sub'],
-                'iat' => $payload['iat'],
+                // 'iat' => $payload['iat'],
```

Comment:
For backwards compatibility, `iat` should be included in the `persistent_claims` config and marked as Breaking Change accordingly

Reply:
@jansgescheit one question: can we add the `iat` in `$persistentClaims` conditionally if it exists on `payload`? 

In that way, it'll not crash if someone wants to use a custom one.

Reply:
For my understanding, the `iat` is always included in the payload. Therefore, not much will be achieved here.

Reply:
So we cannot just remove it right? On that case should be additionally added with a `if` condition if for some reason the user doesn't want to have it in persist claims.

Reply:
That would make sense to avoid a breaking change.  Perhaps via a configuration switch with a denylist for filtering

Reply:
Ok, so let's add the if to it for that we can merge it.

Thanks